packages/core/admin/admin/src/components/GuidedTour/Homepage.tsx

```diff
@@ -5,20 +5,34 @@ import { ArrowRight } from '@strapi/icons';
 import { useIntl } from 'react-intl';
 import { NavLink } from 'react-router-dom';
 
-import { LAYOUT_DATA, States, STATES } from './constants';
+import { SUPER_ADMIN_LAYOUT_DATA, LAYOUT_DATA, States, STATES } from './constants';
 import { Number, VerticalDivider } from './Ornaments';
 
-const GuidedTourHomepage = () => {
-  const { guidedTourState, setSkipped } = useGuidedTour();
+interface GuidedTourHomepageProps {
+  visibility: string;
+}
+
+const GuidedTourHomepage = ({ visibility }: GuidedTourHomepageProps) => {
+  const { guidedTourState, setSkipped, guidedTourVisibility } = useGuidedTour();
   const { formatMessage } = useIntl();
   const { trackUsage } = useTracking();
 
-  const sections = Object.entries(LAYOUT_DATA).map(([key, val]) => ({
+  const layout = visibility === 'super-admin' ? SUPER_ADMIN_LAYOUT_DATA : LAYOUT_DATA;
+  const layoutCopy = JSON.parse(JSON.stringify(layout));
```

Comment:
Why are we doing this?

Reply:
In order to remove the `inviteUser` step. Layout variables are readOnly.

```
 if (process.env.NODE_ENV === 'development') {
    delete layoutCopy.inviteUser;
  }
```

Reply:
It's quite poor for performance and you do it every render, especially since these are constants in TS only, you're also going to break the TS inference, I would instead correct the types so you can delete this safely without performing a clone